<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Renju</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="gameMenu">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="newGameAction"/>
    <addaction name="cancelLastMoveAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="settingsMenu">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="soundAction"/>
   </widget>
   <addaction name="gameMenu"/>
   <addaction name="settingsMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newGameAction">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="cancelLastMoveAction">
   <property name="text">
    <string>Cancel last move</string>
   </property>
  </action>
  <action name="soundAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sound</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Exit </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
